go.mod

```diff
@@ -6,6 +6,8 @@ go 1.22.0
 // that will always use its bundled toolchain.
 toolchain go1.22.5
 
+replace github.com/grafana/alerting => /Users/titolins/dev/alerting
```

Comment:
Leaving this as a reminder to update go.mod once https://github.com/grafana/alerting/pull/247 gets merged